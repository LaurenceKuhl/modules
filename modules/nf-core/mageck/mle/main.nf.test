nextflow_process {

    name "Test Process MAGECK_MLE"
    script "modules/nf-core/mageck/mle/main.nf"
    process "MAGECK_MLE"

    test("Should run without failures") {

        when {
            params {
                // define parameters here. Example:
                // outdir = "tests/results"
            }
            process {
                """
                // define inputs of the process here. Example:
                // input[0] = file("test-file.txt")
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
