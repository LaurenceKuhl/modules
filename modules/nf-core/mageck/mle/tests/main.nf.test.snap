{
    "mageck_mle - csv - stub": {
        "content": [
            [
                "versions.yml:md5,d83f4ee1ef0e972c5ea9a8deee7ba3b3"
            ]
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "24.04.2"
        },
        "timestamp": "2024-06-04T10:58:49.575230952"
    },
    "mageck_mle - csv": {
        "content": [
            [
                "versions.yml:md5,d83f4ee1ef0e972c5ea9a8deee7ba3b3"
            ]
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "24.04.2"
        },
        "timestamp": "2024-06-04T10:55:23.297715246"
    }
}